projects/ad9081_fmca_ebz/vck190/system_project.tcl

```diff
@@ -35,22 +36,23 @@ source $ad_hdl_dir/projects/scripts/adi_board.tcl
 #      make JESD_MODE=64B66B RX_LANE_RATE=24.75 TX_LANE_RATE=24.75 RX_JESD_M=8 RX_JESD_L=8 RX_JESD_S=2 RX_JESD_NP=12 TX_JESD_M=8 TX_JESD_L=8 TX_JESD_S=2 TX_JESD_NP=12
```

Comment:
Could you add some examples for JESD204B? Could be the ones you used to test your changes.
Then these should be in the documentation as well.